.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
       env: GO111MODULE=off
     - go: "1.12"
       env: GO111MODULE=on
-    - go: tip
+    - go: "1.x"
```

Comment:
trying to understand what this does. Does this force picking only a major release version? i.e. 1.12 vs 1.12.4.